src/cba-list-new/cba-list-new.js

```diff
@@ -1,4 +1,4 @@
-import {html, render} from 'lit-html';
+import { html, render } from 'lit-html';
```

Comment:
After changes here https://github.com/browser-automation/cba-components/pull/59 get merged into `master`, we will need to remove top comment and run `npm run test:eslint -- fix` to fix all unrelated changes here.